docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -2,7 +2,7 @@
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.25.0
+ENVTEST_K8S_VERSION = 1.26.0
```

Comment:
We need a PR against the master to update it to 1.26.1 because we bumped the k8s version use and we generate a new envtest asset so it is missing.